lib/katello/tasks/upgrades/3.0/update_subscription_facet_registered_at.rake

```diff
@@ -0,0 +1,15 @@
+namespace :katello do
+  namespace :upgrades do
+    namespace '3.0' do
+      task :update_subscription_facet_registered_at => ["environment"]  do
+        User.current = User.anonymous_api_admin
+        puts _("Updating registered at time for subscription facets")
+
+        Katello::Host::SubscriptionFacet.find_each do |subscription_facet|
+          subscription_facet.update_from_consumer_attributes(subscription_facet.candlepin_consumer.consumer_attributes)
+          subscription_facet.save!
+        end
```

Comment:
It might be worthwhile to catch a 404 and 410 from the candlepin call. and simply skip that loop